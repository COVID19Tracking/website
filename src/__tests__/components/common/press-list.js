import React from 'react'
import renderer from 'react-test-renderer'
import PressList from '../../../components/common/press-list'

const pressListItems = [
  {
    node: {
      id: '2',
      url: 'https://covidtracking.com',
      publication: 'COVID Test publication',
      publishDate: 'April 15, 2020',
    },
  },
  {
    node: {
      id: '1',
      url: 'https://covidtracking.com/two',
      publication: 'COVID Test publication two',
      publishDate: 'April 16, 2020',
    },
  },
]

describe('Components : Common: Press list', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<PressList items={pressListItems} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
