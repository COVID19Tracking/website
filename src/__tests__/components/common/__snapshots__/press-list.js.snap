// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components : Common: Press list renders correctly 1`] = `
<ul
  className="press-list pressList"
>
  <li>
    <cite
      className="title"
    >
      <a
        href="https://covidtracking.com"
      />
    </cite>
    <div
      className="detail-text detailText"
    >
      <span
        className="source"
      >
        COVID Test publication
      </span>
      <span
        className="dotSeparator"
      >
        •
      </span>
      April 15, 2020
    </div>
  </li>
  <li>
    <cite
      className="title"
    >
      <a
        href="https://covidtracking.com/two"
      />
    </cite>
    <div
      className="detail-text detailText"
    >
      <span
        className="source"
      >
        COVID Test publication two
      </span>
      <span
        className="dotSeparator"
      >
        •
      </span>
      April 16, 2020
    </div>
  </li>
</ul>
`;
